docs-2.0/3.ngql-guide/11.edge-statements/4.delete-edge.md

```diff
@@ -0,0 +1,24 @@
+# DELETE EDGE
+
+```ngql
+DELETE EDGE <edge_type> <vid> -> <vid>[@<rank>]
+```
+
+Use `DELETE EDGE` to delete edges. The `DELETE EDGE` statement deletes one vertex or multiple edges at a time. You can use `DELETE EDGE` together with pipe. For more information on pipe, see [Pipe operator](../5.operators/4.pipe.md).
+
+## Examples
+
+```ngql
+nebula> DELETE EDGE serve "player100" -> "team200";
+```
+
+This query deletes the edge whose edge type is serve, source vertex is "player100", destination vertex is "team200".
```

Comment:
the edge whose edge type is serve, source vertex is "player100", destination vertex is "team200". --> the `serve` edge from "player100" to "team200"?

Reply:
> the edge whose edge type is serve, source vertex is "player100", destination vertex is "team200". --> the `serve` edge from "player100" to "team200"?

rewriting is recommended. To keep the sentence grammatically correct, you also can modify your version like this: This query deletes the edge of which the edge type is `serve`, the source vertex is "player100", and the destination vertex is "team200".